test/format/verify.c

```diff
@@ -234,6 +235,20 @@ table_verify_mirror(
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
+    /*
+     * If we are not reading from a checkpoint, start a cursor to pin a page to ensure we never
+     * refresh our snapshot in verification.
+     */
```

Comment:
We probably want to initialize pinned_cursor to NULL here to know if we need to close it later.

Reply:
We don't need to close it explicitly because we close the session in the end.